demo.sh

```diff
@@ -1,2 +1,7 @@
 this is demo.updated data here
-i am adding this for Pull Request
+i am adding this for Pull Request from UAT to master
```

Comment:
remove the from UAT to master
```suggestion
i am adding this for Pull Request from UAT to master
```